RELEASES.md

```diff
@@ -0,0 +1,44 @@
+# Release Process
+
+This section describes the release processes for tracking, preparing, and creating new L3AF releases. This includes information around the release cycles and guides for developers responsible for preparing upcoming stable releases.
+
+## Active development
+
+Active development is happening on the `main` branch, and a new version can be released from it in every 10-12 weeks.
+
+## Release Tracking
+
+Feature work for upcoming releases is tracked through GitHub Projects.
+
+## Release Versions
+
+We would be having a release version for every release. Minor releases are typically designated by incrementing 
+the Y in the version format X.Y.Z. New patch releases for an existing stable version X.Y.Z are published by incrementing
+the Z in the version format.
+
+## Stable releases
+
+Stable releases of L3AF include:
+
+* Extended maintenance window (any version released in the last 6/12 months).
+* Stability fixes backported from the `main` branch (anything that can result in a crash).
+* Bugfixes, deemed worthwhile by the maintainers of stable releases.
```

Comment:
Do "stable releases" have different maintainers from the rest of the project?

Reply:
@dthaler here maintainers are same as code owners.

Reply:
```suggestion
* Bugfixes, deemed worthwhile by the maintainers.
```